README.md

```diff
@@ -1,63 +1,161 @@
 # npl-integrations
 
-## Pre-requisites
-Make sure your `hosts` file includes keycloak at 127.0.0.1:
+The npl-integrations repo contains a sample project that demonstrates how to integrate with [Noumena's Operating Engine](https://documentation.noumenadigital.com/engine/applications/engine/) using
+different programming languages and frameworks.
+It includes a Python listener service, a typescript React web app, and a Python Streamlit UI.
 
-![img.png](docs/img.png)
+The underlying NPL code is an extension of the IOU, which can be found
+in the [npl-starter](https://github.com/NoumenaDigital/npl-starter) repository.
 
 ## Building and running with docker locally
-1. Run `mvn clean install` to build and generate NPL-api and clients.
-2. Run `docker compose up --build -d` to ensure python and frontend are also build, then create and start containers.
 
-URLs:
-* Swagger UI of Engine APIs: http://localhost:12000/swagger-ui/
-* Keycloak admin console: http://localhost:11000
-* Webapp: http://localhost:8090
+The project can be built and run locally using docker-compose.
```

Comment:
```suggestion
The project can be built and run locally using Docker Compose.
```

To avoid any confusion with the deprecated binary (or add .yml if you mean the file itself).

Reply:
Also below